public/store.js

```diff
@@ -1,34 +1,39 @@
 import { atom, useAtom } from "jotai";
 
 const initialTranslations = [
-  { id: 1, word: "verjaardag", language: "Dutch", translated: "Geburtstag" },
-  { id: 2, word: "uitnodigen", language: "Dutch", translated: "einladen" },
+  { id: "1", word: "verjaardag", language: "Dutch", translated: "Geburtstag" },
```

Comment:
why did you need to change your IDs from numbers to strings?

Reply:
Otherwise the form was not recognizing the entry .